lib/zoom/actions/user.rb

```diff
@@ -108,6 +108,12 @@ def user_permissions(*args)
         params.require(:user_id)
         Utils.parse_response self.class.get("/users/#{params[:user_id]}/permissions", headers: request_headers)
       end
+
+      def user_vanity_name(*args)
+        params = Zoom::Params.new(Utils.extract_options!(args))
+        params.require(:vanity_name)
+        Utils.parse_response self.class.get("/users/vanity_name", query: params, headers: request_headers)
```

Comment:
Just to make sure we aren't passing anything else.
```suggestion
        Utils.parse_response self.class.get("/users/vanity_name", query: params.slice(:vanity_name), headers: request_headers)
```

Reply:
good idea